packages/api-request-builder/src/build-query-string.js

```diff
@@ -105,5 +106,7 @@ export default function buildQueryString(
 
   if (version) queryString.push(`version=${version}`)
 
+  if (dataErasure) queryString.push(`dataErasure=${String(dataErasure)}`)
```

Comment:
Using `String()` since linter yells about not coercing booleans otherwise.

`Cannot coerce `dataErasure` to string because boolean [1] should not be coerced.`

Reply:
As a side note: magic boolean arguments are kind of discouraged. 

1. From a call site you never know what it does

Instead of `withErasure(true)` it would be `withEresure({ shouldErase: true })`. 

That obviously makes the API more clunky and shows a slight flaw. Maybe we can omit the argument all together. And hence also the check `if(shouldErease)`?